### ISSUE TITLE
generate mode：repaint

### ISSUE THREAD

**Issue by wwwzzzrrr1013** [yuanzhi-zhu/DiffPIR#38]
Hello author, I am a beginner in image restoration. May I ask if the repaint in generate mode is related to RePaint in "Repainting using Denoising Diffusion Probabilistic Models"? Looking forward to your answer

**Comment by yuanzhi-zhu** [comment]
yes, the generate mode: repaint tries to replicate the _"Repainting using Denoising Diffusion Probabilistic Models"_.
However, our codebase is not a faithful implementation of RePaint.
